misc/python/materialize/scalability/plot/plot.py

```diff
@@ -20,9 +20,11 @@
 
 
 def scatterplot_tps_per_connections(
+    workload_name: str,
     figure: SubFigure,
     df_totals_by_endpoint_name: dict[str, pd.DataFrame],
     include_zero_in_y_axis: bool,
+    include_workload_in_title: bool,
```

Comment:
you can have the default be False here and this no need to set it to False in the majority of the invocations